kms++util/inc/kms++util/resourcemanager.h

```diff
@@ -15,7 +15,9 @@ class ResourceManager
 	Connector* reserve_connector(const std::string& name = "");
 	Connector* reserve_connector(Connector* conn);
 	Crtc* reserve_crtc(Connector* conn);
-	Plane* reserve_plane(Crtc* crtc, Plane::Type type, PixelFormat format = PixelFormat::Undefined);
+	Plane* reserve_plane(Crtc* crtc, Plane::Type typemask, PixelFormat format = PixelFormat::Undefined);
+	Plane* reserve_plane(Crtc* crtc, unsigned typemask, PixelFormat format = PixelFormat::Undefined);
```

Comment:
The uint typemask shouldn't be necessary... I think we can do it with python's enum support. See https://docs.python.org/3.4/library/enum.html and IntEnum for example. How to expose the PlaneType as such from pykms, I don't know yet. But I presume it's possible.